### ISSUE TITLE
Invalid typescript generation for empty schema

### ISSUE THREAD

**Issue by danicc097** [anymaniax/orval#1207]
## What are the steps to reproduce this issue?

Generate spec:

```ts
import { defineConfig } from 'orval'

export default defineConfig({
  main: {
    output: {
      mode: 'tags-split',
      target: './testspec/main.ts',
      schemas: './testspec/model',
      client: 'react-query',
      tsconfig: './tsconfig.json',
    },
    input: {
      target: 'testspec.yaml',
    },
    hooks: {
      afterAllFilesWrite: 'prettier --write',
    },
  },
})
```

```yaml
openapi: 3.0.3
info:
  description: test
  title: test
  version: test
paths:
  /ping:
    get:
      summary: test
      responses:
        '200':
          description: ok
components:
  schemas:
    DbNotificationID: {}
```

## What happens?

Outputs invalid typescript:

```ts
export interface DbNotificationID unknown
```

## What were you expecting to happen?

Same output as in 6.23.0: 

```ts
export interface DbNotificationID = unknown
```

## What versions are you using?

**Package Version:** 6.24.0

**Comment by soartec-lab** [comment]
Thank you made this report and suggest.
Looks good `interface DbNotificationID {}`. you fix It is also possible to create a PR.

**Comment by soartec-lab** [comment]
I'll l submit a fix for this issue, so I assigned it to me.

**Comment by soartec-lab** [comment]
Hi, @danicc097 @dyatko 

Since we released "v6.25.0" this issue has been fixed 👍
